<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInterpolationSetupForm</class>
 <widget class="QFrame" name="CInterpolationSetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="hl_PitchOnGround_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QCheckBox" name="cb_LogInterpolation">
     <property name="text">
      <string>log.interpolation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cb_DebugDriver">
     <property name="text">
      <string>driver dbg. msgs.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cb_EnableParts">
     <property name="text">
      <string>enable parts</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cb_ForceFullInterpolation">
     <property name="text">
      <string>force full interpolation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cb_FixSceneryOffset">
     <property name="toolTip">
      <string>experimental feature to fix offset values</string>
     </property>
     <property name="text">
      <string>fix scenery offset (exp.)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="cb_SendGndFlagToSim">
     <property name="text">
      <string>send gnd.to simulator</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="fr_PitchOnGround">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="hl_PitchOnGround">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_PitchOnGround">
        <property name="text">
         <string>pitch on gnd.:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_PitchOnGround">
        <property name="placeholderText">
         <string>pitch on gnd.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="wi_InterpolatorType" native="true">
     <layout class="QHBoxLayout" name="hl_InterpolatorType">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_Spline">
        <property name="text">
         <string>spline</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_Linear">
        <property name="text">
         <string>linear</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_InterpolatorType">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rb_Spline</tabstop>
  <tabstop>rb_Linear</tabstop>
  <tabstop>le_PitchOnGround</tabstop>
  <tabstop>cb_EnableParts</tabstop>
  <tabstop>cb_SendGndFlagToSim</tabstop>
  <tabstop>cb_FixSceneryOffset</tabstop>
  <tabstop>cb_ForceFullInterpolation</tabstop>
  <tabstop>cb_LogInterpolation</tabstop>
  <tabstop>cb_DebugDriver</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
